sonic-xcvrd/xcvrd/xcvrd_utilities/port_mapping.py

```diff
@@ -83,21 +88,28 @@ def logical_port_name_to_physical_port_list(self, port_name):
             else:
                 return None
 
+def subscribe_port_config_change(db_list=['CONFIG_DB']):
+    port_tbl_map = {
+        'APPL_DB': swsscommon.APP_PORT_TABLE_NAME,
```

Comment:
We'll listen to APPL_DB for the PORT CONFIG changes, and STATE_DB for XCVR INSERTION/REMOVAL

Reply:
The trick is, not all the fields in APPL_DB will be updated to STATE_DB by the orchagent as of now, it's part of synchronization issue between syncd and orchagent, it's outside the scope of this PR